packages/block-editor/src/components/off-canvas-editor/block.js

```diff
@@ -132,6 +133,14 @@ function ListViewBlock( {
 		  )
 		: __( 'Options' );
 
+	const editAriaLabel = blockInformation
```

Comment:
Why not put this in the component itself?

Reply:
Seems like other labels are here as well.